src/AutoRest.CSharp/Common/Generation/LongRunningOperationWriter.cs

```diff
@@ -103,7 +111,7 @@ void WriteResultFunction(bool async)
 
                     using (writer.Scope())
                     {
-                        writer.Line($"_operation = new {helperType}(this, clientDiagnostics, pipeline, request, response, {typeof(OperationFinalStateVia)}.{operation.FinalStateVia}, {operation.Diagnostics.ScopeName:L});");
+                        writer.Line($"_operation = new {helperType}{paramStart}clientDiagnostics, pipeline, request, response, {typeof(OperationFinalStateVia)}.{operation.FinalStateVia}, {operation.Diagnostics.ScopeName:L});");
```

Comment:
You can split the `Line` call into multiple `Append` and have the `                var paramStart = isGenericOperation ? "(this, " : "(";` logic inline.

```C#
if (isGenericOperation)
{
writer.Append("this,");
}